core/engine_test.go

```diff
@@ -720,6 +720,7 @@ func TestSentReceivedMetrics(t *testing.T) {
 	})
 }
 
+// TODO: delete, functionality duplicated in cmd/integration_test.go
```

Comment:
Here and in other places marked with such TODOs? Why should we keep it, then? :thinking:

Reply:
I kind of wanted to delete the old `Engine` tests in the PR that removes the whole `Engine`, the one that closes https://github.com/grafana/k6/issues/1889. 

Since the integration tests and the ones they replace in `core/engine_test.go` are not _exactly_ perfectly aligned, I don't see the harm in keeping the old ones until then, for a tiny bit of extra assurance. I plan to submit at least 2 more PRs before I submit the one with the `Engine` removal, one for v0.42.0 with another bugfix, and another for the start of v0.43.0, and https://github.com/grafana/k6/pull/2787 already exists for v0.43.0 merging, so having these extra tests for the current behavior until then seems like a good idea.

Reply:
Okay, then I believe we need to improve the comment again because what you're saying makes sense, but it's not clear from the current comment:
 
```
// TODO: delete, functionality duplicated in cmd/integration_test.go
```

From my point of view, even the references to the PR/issues could improve it.

```
// TODO: delete (#1889), functionality duplicated in cmd/integration_test.go
```

Reply:
Resolved by https://github.com/grafana/k6/pull/2795/commits/3e73e8783b13e7f82e3ce23a3c7b907f2e536e72